dev/cnf/oss_dependencies.maven

```diff
@@ -518,6 +518,16 @@ org.springframework:spring-webmvc:4.3.13.RELEASE
 org.springframework:spring-webmvc:5.0.8.RELEASE
 org.springframework:spring-webmvc:5.1.4.RELEASE
 org.springframework:spring-websocket:5.0.8.RELEASE
+org.springframework.security:spring-security-web:5.2.2.RELEASE
```

Comment:
please keep this file in alphabetical order so that we can avoid duplicates where possible

Reply:
in fact, there are already multiple versions of these dependencies ... please check if 5 different versions of spring-core (for example) are really needed?

Reply:
Thank you. I checked all the dependencies starting with "org.springframework". And found that there are 2 dependencies not used 
"org.springframework:spring-webmvc:4.3.13.RELEASE", 
"rg.springframework:spring-webflux:5.1.4.RELEASE"
And have make a commit to delete them. The other dependencies are used somewhere(we have several springboot app versions).